esiclient/utils.py

```diff
@@ -39,7 +39,7 @@ def get_network_info_from_port(port, client):
     return get_network_display_name(network), fixed_ip
 
 
-def get_full_network_info_from_port(port, client):
+def get_full_network_info_from_port(port, client, get_trunk_details=False):
```

Comment:
Apologies - I thought the get_trunk_details switch was called from the same function, but they're not. That makes me think that this would actually be cleaner split into two functions, with the trunk details calling this one.

Reply:
Fixed! Also tested on pilot deployment with command "openstack esi node network attach oct4-10 --trunk naved-created-trunk". It successfully attached esi-naved-created-trunk-provisioning-trunk-port to oct4-10.